op_crates/web/03_global_interfaces.js

```diff
@@ -4,7 +4,7 @@
   const illegalConstructorKey = Symbol("illegalConstuctorKey");
 
   class Window extends EventTarget {
-    constructor(key) {
+    constructor(key = null) {
       if (key !== illegalConstructorKey) {
         throw new TypeError("Illegal constructor.");
       }
```

Comment:
We got a couple more such constructors, @benjamingr do you want to check all of them in a single PR?

Reply:
Yes if a single PR is easier to review I'll push all the changes in a single PR :]

Reply:
I've put all the changes here (in this PR) and verified in the spec that this fix is indeed correct (and required) for it.